src/DssDirectDepositHub.sol

```diff
@@ -223,7 +213,7 @@ contract DssDirectDepositHub {
     }
 
     /**
-        @notice remove nope on daiJoin to kill hub
+        @notice remove hope on daiJoin to kill hub
```

Comment:
We might want to revert this change and evaluate this function in another PR. I think this is useless. Removing the `vat` access to the `daiJoin` should not be really needed and we should avoid adding non useful functions.